### PR TITLE
User search api autocomplete for Event Search Created by User

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description

 Event Search fields "Event created by user" and "Event updated by user" should do Ajax like API queries for autocomplete.
## Tickets

* [CNFT1-1763](https://cdc-nbs.atlassian.net/browse/CNFT1-1763)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

[CNFT1-1763]: https://cdc-nbs.atlassian.net/browse/CNFT1-1763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mdavis-enquizit** [comment]
Screenshot 2024-01-26 at 1 37 17 PM

**Comment by mdavis-enquizit** [comment]
Screenshot 2024-01-26 at 1 46 43 PM

**Comment by mdavis-enquizit** [comment]
Screenshot 2024-01-26 at 1 53 43 PM

File: apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationGeneralFields.tsx [hunk 1]
```diff
@@ -15,13 +14,13 @@ import { SearchCriteriaContext } from 'providers/SearchCriteriaContext';
 import { ChangeEvent, ReactElement } from 'react';
 import { Controller, UseFormReturn, useWatch } from 'react-hook-form';
 import { formatInterfaceString } from 'utils/util';
+import { UserAutocomplete } from 'options/autocompete/UserAutocomplete';
```
Comment (mdavis-enquizit) [inline apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationGeneralFields.tsx:18]: Name should be changed so it doesn't clash potential class or confuse with `public static userAutocomplete`

**Review by adamloup-enquizit — APPROVED** [review]
Nice job!

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**